projects/canopy-test-app/src/app/app.component.html

```diff
@@ -641,6 +641,13 @@ <h3>Carousel item 3</h3>
               lgMarginTop="xxl"
               size="sm"
             ></lg-spinner>
+
+            <lg-spinner
+            variant="color"
+            text="Extra Small spinner"
+            lgMarginTop="xxl"
+            size="xs"
+          ></lg-spinner>
```

Comment:
It looks like the demo app isn't covered by the auto formatter :) 
This needs a bit of love with the indentation.

```suggestion
            <lg-spinner
              variant="color"
              text="Extra Small spinner"
              lgMarginTop="xxl"
              size="xs"
            ></lg-spinner>
```

Reply:
Fixed, thanks!